src/helpers/reset.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import path from 'path'
+import rimraf from 'rimraf'
+import resolveUserDataDirectory from 'helpers/resolveUserDataDirectory'
+import { disable as disableDBMiddleware } from 'middlewares/db'
+import db from 'helpers/db'
+import { delay } from 'helpers/promise'
+
+function resetLibcoreDatabase() {
+  console.log(`PREP`)
+  const dbpath = path.resolve(resolveUserDataDirectory(), 'sqlite/')
+  console.log(`Deleting ${dbpath}`)
+  rimraf.sync(dbpath, { glob: false })
+}
+
+export async function hardReset() {
+  resetLibcoreDatabase()
+  disableDBMiddleware()
+  db.resetAll()
+  await delay(500)
+  window.location.href = ''
```

Comment:
I know it's legacy moving code, but let's have a single way to reload the page :D
(softReset use electron currentWindow, so we can use either the one or the other)